src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const lines = sourceString
+    .split(';')
+    .map((line) => line.trim())
+    .filter((line) => line.length > 0);
+
+  lines.forEach((line) => {
+    const [property, value] = line.split(':').map((part) => part.trim());
+
+    if (property && value !== undefined) {
```

Comment:
```suggestion
    if (property && value) {
```
You don't need to compare with undefined, just check if a value exists